ahfe_tutorial/ahfe_tutorial.ipynb

```diff
@@ -815,46 +837,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 19,
    "id": "be690ef1-3243-4114-b56f-5babbb660af6",
    "metadata": {
     "scrolled": true
    },
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "/Users/hannahbaumann/openfe/openfe/protocols/openmm_rfe/_rfe_utils/compute.py:56: UserWarning: Non-GPU platform selected: CPU, this may significantly impact simulation performance\n",
-      "  warnings.warn(wmsg)\n",
-      "WARNING:root:Non-GPU platform selected: CPU, this may significantly impact simulation performance\n",
-      "WARNING:openmmtools.multistate.multistatereporter:Warning: The openmmtools.multistate API is experimental and may change in future releases\n",
-      "WARNING:root:Non-GPU platform selected: CPU, this may significantly impact simulation performance\n",
-      "WARNING:openmmtools.multistate.multistatesampler:Warning: The openmmtools.multistate API is experimental and may change in future releases\n"
-     ]
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Please cite the following:\n",
```

Comment:
It's not a blocker, but as a user I generally prefer having this output there if possible. It helps me know that what is being written out is what should be expected.

Reply:
Sounds good - I generally agree but this was borderline too verbose. I think you're right in this instance because these warnings might confuse people otherwise.

Reply:
Oh I take it back - I intentionally commented out this block to avoid having a simulation run unless the user chooses to. This matches the behavior in the other notebooks.